M2/Macaulay2/packages/VectorGraphics.m2

```diff
@@ -719,7 +719,8 @@ new SVG from GraphicsObject := (S,g) -> (
     ss
     )
 
-html GraphicsObject := g -> html SVG g;
+hypertext GraphicsObject := g -> SVG g
```

Comment:
Which objects should have a `hypertext` method now?

Reply:
this is completely optional -- basically, for types whose `html` output is going through some Hypertext stuff, instead of writing
```
html MyType := x -> html ( DIV x)
```
or whatever, you can write
```
hypertext MyType := x -> DIV x
html MyType := html @@ hypertext
``
cleaner and potentially useful long term.

Reply:
Not sure if I follow. Where would `hypertext MyType` be called from?

Reply:
at this stage, only by `html`.

Reply:
Still confused, but with the fix I suggested below (`html Thing := x -> if lookup(hypertext, class x) =!= null then html hypertext x ...`) would eliminate the need for all `html Blah := html @@ hypertext`.